### DISCUSSION TITLE
'warning: TLS library problem' -> with messages sent by Outlook

### DISCUSSION THREAD

**Discussion by j75** [openssl/openssl#22928] [Q&A]
In my Postfix server logs, on Debian 12, I encounter the following messages:
```
postfix/smtpd[1920086]: warning: TLS library problem: error:0A000102:SSL routines::unsupported protocol:../ssl/statem/statem_srvr.c:1657:
postfix/smtpd[1920086]: lost connection after STARTTLS from mail-db8eur05olkn2025.outbound.protection.outlook.com[40.92.89.25]
```
Beside Outlook, the same error is provoked by Windows 11 Courier program; the OpenSSL version is `3.0.11 19 Sep 2023`.

My questions are:
* how could I exactly see what unsupported protocol is used?
* how this error could be avoided?

**Comment by vdukhovni** [comment]
This question should be asked on the [postfix-users](https://list.sys4.de/postorius/lists/postfix-users.postfix.org/) list, rather than here.

NOTE: Generally, configuration-specific questions such as this also require that the OP post the configuration settings, as described in the Postfix [DEBUG_README](https://www.postfix.org/DEBUG_README.html#mail). Specifically, and without changing whitespace including line-breaks, include with the question the outputs of:

* postconf -nf
* postconf -Mf
